test/hotspot/jtreg/serviceability/jvmti/SetBreakpoint/libTestManyBreakpoints.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2013, 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+#include <string.h>
+#include <stdio.h>
+
+#include "jvmti.h"
+
+#define SUBCLASS_NAME "Lserviceability/jvmti/SetBreakpoint/TestManyBreakpoints$Target"
+
+static jvmtiEnv *jvmti = NULL;
+
+void JNICALL classprepare(jvmtiEnv* jvmti_env, JNIEnv* jni_env, jthread thread, jclass klass) {
+    char* buf;
+    (*jvmti)->GetClassSignature(jvmti, klass, &buf, NULL);
```

Comment:
It is required to check jvmti error status for every function. So test fails early.
There is no common library so far so you could just copy-paste 'check_jvmti_status' from other tests.

Reply:
Done.